woocommerce/woocommerce
# Put back the remove element for multiselects on selectWoo

### All Submissions:






### Changes proposed in this Pull Request:



Closes #26845.

### How to test the changes in this Pull Request:

1. Create 2 products, one named sth like `Product with a <a href="https://www.google.sk">link</a>`.
2. Go to the other product and try to set the first one as linked product.
3. In master you should see everything escaped, but no `x` to remove 
4. Apply this PR
5. See `x` to remove linked product. Same with attributes, etc.

### Other information:




### Changelog entry

> Fix - Put back the remove element for multiselects on selectWoo.
